packages/network-controller/src/NetworkController.test.ts

```diff
@@ -3722,6 +3724,108 @@ describe('NetworkController', () => {
       });
     });
   });
+
+  describe('Network Configurations', () => {
+    it('should add a network configuration', async () => {
```

Comment:
In rewriting these tests recently, I dropped the word "should" from the test names. I actually don't know whether we have a convention around this or not. It's possible we do and if so I'm happy to submit a new PR changing these test names. But in the meantime, at least in the context of this file, they aren't being used. What do you think about using "adds a network configuration"? Same for the other tests.

Reply:
I guess I don't have a good reason for this other than it's shorter to say.

Don't worry about this — I'll update these other tests to match.

Reply:
I hadn't realized it until you brought this up, but now I'm noticing that I am inconsistent about whether I follow this convention. Agreed that this isn't a concern for this PR but perhaps it is worth deciding whether this style is recommended or not.